libs/datamodel/core/tests/reformat/mod.rs

```diff
@@ -1,2 +1,831 @@
-pub mod reformat;
```

Comment:
So far our pattern has been to keep the mods empty. Maybe it's a better option to just rename it? Or why is clippy complaining about this?

Reply:
This is the module_inception lint mentioned above; perhaps renaming is indeed better. Will think about it!